### DISCUSSION TITLE
Exclude Category from Mark All as Read

### DISCUSSION THREAD

**Discussion by jsphnecclia** [FreshRSS/FreshRSS#5399] [Q&A]
Is it possible to exclude a category from Mark All as Read?

Say I have a group of feeds (category) that I do not want to ever miss _any_ of. And a ton of other feeds that I need to mark as read. Is it possible to do this? 

_Or_, is there a solution to this problem other than excluding a category?

**Comment by Alkarex** [comment]
This cannot be done at category level for now, but one approach is to not show some feeds in the main stream, and only show them in their category.
This way, the feeds are neither shown nor marked as read from the main stream.

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/624e2ab2-05f2-4f87-9f99-fbb6e62e528c)
